onnxruntime/test/util/test_environment.cc

```diff
@@ -21,8 +22,12 @@ namespace test {
 
 static std::unique_ptr<::onnxruntime::logging::LoggingManager> s_default_logging_manager;
 
+const ::onnxruntime::Environment& GetEnvironment() {
+  return ((OrtEnv*)*ort_env.get())->GetEnvironment();
+}
```

Comment:
Oh my. What a cryptic syntax we have created. Is this used in technical interviews to confuse candidates? :)